packages/cli/openapi-parser/src/openapi/v3/extensions/getFernExamples.ts

```diff
@@ -2,14 +2,15 @@ import { EndpointExample, FernOpenapiIr } from "@fern-api/openapi-ir-sdk";
 import { RawSchemas } from "@fern-api/yaml-schema";
 import { OpenAPIV3 } from "openapi-types";
 import { getExtension } from "../../../getExtension";
+import { OpenAPIExtension } from "./extensions";
 import { FernOpenAPIExtension } from "./fernExtensions";
 
 export function getFernExamples(operationObject: OpenAPIV3.OperationObject): EndpointExample[] {
```

Comment:
I believe this is called `getFernExamples` because it was meant for the fern extension specifically -- how about `getExamplesFromExtension` ?